plugins/source/aws/resources/services/s3/buckets.go

```diff
@@ -17,10 +16,11 @@ import (
 func Buckets() *schema.Table {
 	tableName := "aws_s3_buckets"
 	return &schema.Table{
-		Name:      "aws_s3_buckets",
-		Resolver:  fetchS3Buckets,
-		Transform: transformers.TransformWithStruct(&models.WrappedBucket{}),
-		Multiplex: client.AccountMultiplex(tableName),
+		Name:                "aws_s3_buckets",
+		Resolver:            listS3Buckets,
+		PreResourceResolver: resolveS3BucketsAttributes,
```

Comment:
This wasn't using a `PreResourceResolver` before, so think this may slightly change the behavior. Should it still be considered a refactor? Maybe `fix` would be more appropriate.

Reply:
Fair.